File: check_dossiers.mjs
import { Client } from 'pg';

async function check() {
    const client = new Client({
        connectionString: 'postgresql://postgres:kadrilex-secure-db-2026@37.59.99.86:5432/postgres'
    });
    
    await client.connect();
    
    try {
        const res = await client.query('SELECT id, numero, titre FROM "Dossier" ORDER BY "numero" ASC LIMIT 20');
        console.log("Top 20 Dossiers:");
        console.table(res.rows);
    } catch (e) {
        console.error(e);
    } finally {
        await client.end();
    }
}

check();

File: check_dossiers.py
import psycopg2

try:
    conn = psycopg2.connect("postgresql://postgres:kadrilex-secure-db-2026@37.59.99.86:5432/postgres")
    cur = conn.cursor()
    
    cur.execute('SELECT "id", "numero", "titre", "clientId" FROM "Dossier" ORDER BY "numero" ASC LIMIT 20')
    rows = cur.fetchall()
    
    for row in rows:
        print(row)
except Exception as e:
    print(e)
finally:
    if 'conn' in locals():
        conn.close()
